cannon/mipsevm/mips.go

```diff
@@ -75,107 +55,22 @@ func (m *InstrumentedState) handleSyscall() error {
 		m.state.ExitCode = uint8(a0)
 		return nil
```

Comment:
Just a note that I did not make helpers for some of the simpler logic here (sysBrk, sysClone, sysExitGroup).   Leaving sysClone inline definitely makes sense as it is a noop implementation and we'll have a real implementation in MTCannon.